demo/src/App.js

```diff
@@ -172,13 +195,45 @@ function App() {
           </Button>
           <Button
             disabled={!isPyramid || isLoading}
-            onClick={toggleOverview}
+            onClick={toggleLinkedView}
             variant="outlined"
             size="small"
             fullWidth
           >
-            {overviewOn ? 'Hide' : 'Show'} Picture-In-Picture
+            {useLinkedView ? 'Overview' : 'Linked'}
```

Comment:
Ok, how does this work?

App.js:
`Hide/Show Picture-in-Picture` --> `Hide/Show Overview`
`Overview` --> `Picture-in-Picture`
`Linked` --> `Side-by-side`

Classes:
`OverviewDetailViewer` --> `PictureInPictureViewer`
`LinkedDetailViewer` --> `SideBySideViewer`

Thoughts?  Does this work?

Reply:
Hmm. This is still confusing to me. There is a button that toggles between `hide/show picture-in-picture` and the same behavior is observed with the `overview` button when views are linked. I would prefer two toggles at the high level that are mutually exclusive :

`show/hide picture-in-picture` and `show/hide side-by-side`. If one of these buttons is on, the other is disabled. If `hide side-by-side` (so `SideBySideViewer` is in use), then conditionally render the lock pan / lock zoom buttons. If the user wants to try out the other view, they must first go through VivViewer. This will make it so the end-user doesn't expect the side-by-side view to also be able to show a picture-in-picture when in use.